src/app/[locale]/_component/sessionActiveComp/sessionActiveComp.tsx

```diff
@@ -36,20 +40,11 @@ const SessionExistingComp = ({ title, showArrowBackBtn }: SessionProps): React.R
         <Grid item xs={12} justifySelf={'center'}>
           <Introduction
             title={title}
-            summary={
-              <>
-                <span>
-                  {t.rich('lplogoutpostlogin.activesession', {
-                    deviceModel: 'iPhone 12 Pro',
-                    strong: (chunks) => <strong>{chunks}</strong>,
-                  })}
-                </span>
-              </>
-            }
+            summary={`Al momento hai una sessione attiva sull’app IO, valida fino al ${expirationDate.toLocaleDateString()}. Se hai perso il dispositivo, termina la sessione. Potrai poi accedere nuovamente da qualsiasi altro dispositivo.`}
             summaryColumns={{ xs: 12, md: 6 }}
           />
         </Grid>
-
+        {/*
```

Comment:
Do we have a JIRA ticket to put along with this comment?

Reply:
This is the task related to the creation of the page in figma for this case: https://pagopa.atlassian.net/browse/IOPID-710

Reply:
We usually put this reference in the comment itself:

```suggestion
        {/* FIXME: https://pagopa.atlassian.net/browse/IOPID-710
```